index.html

```diff
@@ -1206,7 +1206,7 @@ <h3>Errors</h3>
   <td><code>no such cookie</code>
   <td>No cookie matching the given path name
    was found amongst the <a>associated cookies</a>
-   of the <a>current browsing context</a>’s <a>active document</a>.
+   of the <a>current browsing context</a>’s [=navigable/active document=].
```

Comment:
Not sure this is a great change? A navigable's active document is very different from a browsing context's active document, so this just seems misleading?

Reply:
yeah... sorry, I grabbed the only one that was [exported](https://respec.org/xref/?term=active+document)